docs/docs/quickstart.md

```diff
@@ -3,10 +3,14 @@ import TabItem from '@theme/TabItem';
 
 # Track LLM inputs & outputs
 
-:::tip
-For a limited time, the new W&B Inference service is included in your free tier. W&B Inference provides access to leading open-source foundation models via API and the Weave [Playground](./guides/tools/playground.md). 
-- [Developer documentation](./guides/integrations/inference.md)
-- [Product page](https://wandb.ai/site/inference) 
+:::info[💡 New to Weave? Consider starting with W&B Inference]
```

Comment:
Not sure about this block of changes.

Reply:
Yep, we didn't go this way; see the state of `master`